source/code/plugins/oms_common.rb

```diff
@@ -478,6 +480,141 @@ class Common
     @@tzRightFolder = 'right/'
 
     class << self
+
+      # Internal methods
+      # (left public for easy testing, though protected may be better later)
+
+      def clean_hostname_string(hnBuffer)
+        s = hnBuffer.strip
+        # Take off any port numbers:
+        if /\A(\h{4}:\h{4}:\h{4}:\h{4}:\h{4}:\h{4}:\h{4}:\h{4}):\d+\z/ =~ s
```

Comment:
Given that this regex is very similar to the one used in `is_like_ipv6_string()`, we should try to unify them. Can we make it a constant regex with the matching group and use it in both?
Otherwise, at least add a comment that this regex is intended to match an IPv6 address; it is not intuitive.

Reply:
Yes, I agree.  I was focused on a working draft and I kept dropping that task.  Good catch.